borg/crypto.pyx

```diff
@@ -103,7 +120,7 @@ cdef class AES:
         if not out:
             raise MemoryError
         try:
-            if not EVP_EncryptUpdate(&self.ctx, out, &outl, data, inl):
```

Comment:
The point of this changeset is that en/decrypt accept memory views now (and bytes, bytearrays etc. like before), so that the stuff in AESKeyBase doesn't need to copy the payload.